frontend/packages/eslint-plugin-console/lib/config/rules/typescript.js

```diff
@@ -74,14 +74,17 @@ module.exports = {
   //   },
   // ],
 
+  // TODO This rule wasn't working as expected prior to @typescript-eslint/eslint-plugin v2.11.0.
+  // Disabled for now as it is causing failures
   // Enforces the use of as Type assertions instead of <Type> assertions
-  '@typescript-eslint/consistent-type-assertions': [
-    'error',
-    {
-      assertionStyle: 'as',
-      objectLiteralTypeAssertions: 'allow-as-parameter',
-    },
-  ],
+  // '@typescript-eslint/consistent-type-assertions': [
+  //   'error',
+  //   {
+  //     assertionStyle: 'as',
+  //     objectLiteralTypeAssertions: 'allow-as-parameter',
+  //   },
+  // ],
+  '@typescript-eslint/consistent-type-assertions': 'off',
```

Comment:
We one (or more) issues tracking a followup to review these rules and enable them according They should be addressed ASAP to prevent devs from creating code that fails these changes. 
Especially bad for rules like `@typescript-eslint/consistent-type-assertions`.

Reply:
I'll create a Jira task to review these. We can add sub-tasks for individual rules if we want to.

Reply:
https://issues.redhat.com/browse/CONSOLE-1973

Reply:
Added 1 sub-task based on the above comment https://issues.redhat.com/browse/CONSOLE-1975.